cpmonitor/fixtures/example_fixture.json

```diff
@@ -0,0 +1,1099 @@
+[
```

Comment:
I guess, this is here by accident due to the example command in the README?

Reply:
Correct, I thought I deleted it again after figuring that out but seems I forgot to, now done ^^